packages/modular-scripts/src/config/jest/index.ts

```diff
@@ -0,0 +1,116 @@
+import * as fs from 'fs-extra';
+import path from 'path';
+import chalk from 'chalk';
+import type { Config } from '@jest/types';
+import { getCracoJestConfig } from '../index';
+import getModularRoot from '../../utils/getModularRoot';
+import { ModularPackageJson } from '../../utils/isModularType';
+
+const modularRoot = getModularRoot();
+
+const rootPackageJson = fs.readJSONSync(
```

Comment:
Avoid these being in the top level import since they'll run for all commands which will slow startup. Especially blocking IO operations like ```existsSync```.

Reply:
Good catch